Sentinel/Classes/Core/Internal/SentinelListViewModel.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  SentinelListViewModel.swift
+//  Pods
+//
+//  Created by Zvonimir Medak on 28.02.2025..
+//
+
+import Combine
+
+final class SentinelListViewModel: ObservableObject {
+
+    // MARK: - Internal properties
+
+    @Published var serachText: String = ""
+    @Published var sections: [ToolTableSection]
+    let initialSections: [ToolTableSection]
```

Comment:
From the calling side you will see `initialSections` which may create a confusion - what type of sections there could also be. I would rather use `sections` if needed, and internally we can do different kind of "magic" with those.

Or, maybe better, update the `init` method with proper parameter name, but in that case, the property `initialSections` should not be exposed due to potential confusion (it is not used outside of the object).

In short;
* Make `initialSections` private
* Change the parameter name in the `init` method to `sections`